src/objects/page/page.scss

```diff
@@ -25,10 +27,25 @@
   grid-row: 1;
 }
 
-.o-page__content {
+/**
+ * An optional `alert` block can be used to pass an Alert component
+ *
+ * 1. The alert should overlap the content above slightly
+ */
+
+.o-page__alert {
+  @include spacing.fluid-padding-inline;
+  display: grid;
   grid-row: 2;
+  inset-block-start: -1.5em; // 1
```

Comment:
Sorry I didn't notice this before, but can this be a design token, ideally derived from our modular scale (I believe steps 1 or 2 would be close enough)? I notice there's a `size.$overlap-small` token, maybe this can be `size.$overlap-large`?

Reply:
@tylersticka Done! 

https://github.com/cloudfour/cloudfour.com-patterns/pull/1921/commits/beb03f1a019f3b0b2e43ef91d2a5d579dbea7ef1